common/hooks/post-install/11-pkglint-elf-in-usrshare.sh

```diff
@@ -18,7 +18,11 @@ hook() {
         case "${mime}" in
             # Note application/x-executable is missing which is present in most Electron apps
             application/x-sharedlib*|application/x-pie-executable*)
-                matches+=" ${file#$PKGDESTDIR}" ;;
+                if ! [[ ${ignore_in_usr_share} =~ $(basename ${file#$PKGDESTDIR}) ]]
```

Comment:
A test like
```
[[ ${ignore_in_usr_share} != *$(basename "$I")* ]]
```
to the match operator would be consistent with with match tests in the build-helpers.

Regardless, the argument to `basename` should be quoted in case the file has spaces in the name.

Also, why bother stripping out `PKGDESTDIR` from `file` if we are feeding it to basename anyway?